documentation/docs/libraries/java/getting_started.md

```diff
@@ -7,102 +7,110 @@ keywords:
 - jar
 - maven
 - environment variable
+- reference
 ---
 # Getting Started with Java
 
-## Prerequisite
+## Prerequisites
 
 To use the library, we recommend you update Rust to the latest stable version [`$ rustup update stable`](https://github.com/rust-lang/rustup.rs#keeping-rust-up-to-date). Nightly should be fine but some changes might not be compatible.
 
-Ensure you have first installed the required dependencies for the library [here](https://github.com/iotaledger/wallet.rs/blob/dev/README.md). Then also install the following programs:
+Ensure you have installed the [required dependencies for the library](https://github.com/iotaledger/wallet.rs/blob/dev/README.md) first. Then, you can also install the following programs:
 
-- Java & JDK (Make sure $JAVA_HOME env variable) is set
-- [Gradle](https://gradle.org/install/) v4 or higher or [Maven](https://maven.apache.org/download.cgi) installe
-- Cargo ndk (`cargo install cargo-ndk`)
-- Cargo fmt (`rustup component add rustfmt`)
+- Java & JDK (Make sure $JAVA_HOME env variable is set).
+- [Gradle](https://gradle.org/install/) v4 or higher or [Maven](https://maven.apache.org/download.cgi).
+- Cargo ndk (`cargo install cargo-ndk`).
+- Cargo fmt (`rustup component add rustfmt`).
 
 
-Download or clone the `wallet.rs` repository
+Download or clone the `wallet.rs` repository:
+
 ```
 $ git clone https://github.com/iotaledger/iota.rs.git
```

Comment:
do we need the $ here?

Reply:
I think its a preference thing; removing for now.